index.js

```diff
@@ -1,16 +1,17 @@
 var through = require('through');
-var riotCompiler = require('riot/compiler');
+var riotCompiler = require('riot/compiler/compile');
 
 module.exports = function (file, o) {
   var opts = o;
   var content = '';
+  var reqriot = 'var riot = require(\'riot\');';
```

Comment:
Why is this required? Shouldn't the end user add this to the input files if desired?